source/includes/gcp/_vpn_gateways.md

```diff
@@ -1 +1,186 @@
+      ],
+      "kind": "compute#targetVpnGateway",
+      "externalIp": {},
+      "tunnelsName": [],
+      "reserveStaticIP": false,
+      "id": "3112849056897469664",
+      "name": "test-vpn-gw",
+      "shortRegion": "us-central1"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/vpngateways</code>
+
+Retrieve a list of all VPN gateways in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format.
+`description`<br/>*string* | An optional description
+`externalIp`<br/>*Object* | The external IP attached to this VPN gateway.
+`forwardingRules`<br/>Array | List of the forwarding rules which are defined for this VPN gateway
+`id`<br/>*UUID* | Unique identifier for this resource.
+`name`<br/>*string* | The display name of the VPN gateway.
+`network`<br/>*string* | URL of the network to which this VPN gateway is attached.
+`region`<br/>*string* | The URL of the region where the VPN gateway is.
+`status`<br/>*string* | The status of the VPN gateway. One of the following: READY, CREATING, FAILED and DELETING.
+`selfLink`<br/>*string* | Server-defined URL for the resource.
+
+<!-------------------- RETRIEVE A VPN GATEWAY -------------------->
+
+#### Retrieve a VPN gateway
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/vpngateways/3112849056897469664"
+
+# The above command returns JSON structured like this:
+```
+
+```json
+{
+  "data": {
+    "creationTimestamp": "2019-08-27T06:37:19.149-07:00",
+    "description": "",
+    "region": "https://www.googleapis.com/compute/v1/projects/test-area/regions/us-central1",
+    "network": "https://www.googleapis.com/compute/v1/projects/test-area/global/networks/default",
+    "status": "READY",
+    "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/regions/us-central1/targetVpnGateways/test-vpn-gw",
+    "forwardingRules": [
+        "https://www.googleapis.com/compute/v1/projects/test-area/regions/us-central1/forwardingRules/test-vpn-gw-rule-udp500",
+        "https://www.googleapis.com/compute/v1/projects/test-area/regions/us-central1/forwardingRules/test-vpn-gw-rule-udp4500",
+        "https://www.googleapis.com/compute/v1/projects/test-area/regions/us-central1/forwardingRules/test-vpn-gw-rule-esp"
+    ],
+    "kind": "compute#targetVpnGateway",
+    "externalIp": {},
+    "tunnelsName": [],
+    "reserveStaticIP": false,
+    "id": "3112849056897469664",
+    "name": "test-vpn-gw",
+    "shortRegion": "us-central1"
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/vpngateways/:id</code>
```

Comment:
seeing weird error in github but looks to be working.

Reply:
I'll check the linting, could be spaces or weirdo stuff. Thanks.

Reply:
Change pushed.